applications/stream-applications-core/pom.xml

```diff
@@ -17,7 +17,7 @@
 
     <properties>
         <stream-apps-core.version>6.0.0-SNAPSHOT</stream-apps-core.version>
-        <java-functions.version>5.0.1</java-functions.version>
+        <java-functions.version>5.1.0-SNAPSHOT</java-functions.version>
```

Comment:
Looks like this PR is also fixing https://github.com/spring-cloud/stream-applications/issues/603.
Would you mind to mention that as a `Fixes: #603` sentence in the commit message?